lib/features/home/view/home_view.dart

```diff
@@ -1,86 +1,141 @@
+    } catch (e) {}
+    super.dispose();
+  }
+
+  void onShakeToAction(BuildContext context, AppConfigState state) {
+    if (state is DebugOptionEnabledState && state.debugOptionEnabled == true) {
+      // Only listen ShakeDetector when debugOptionTrue
+      detector = ShakeDetector.waitForStart(
+          onPhoneShake: () async {
+            // To limit that shaking will happend more than one so there will be muliple push happened
+            if (detector.mShakeCount == 1) {
+            Navigator.push(context, MaterialPageRoute(builder: (context) {
+                // detector.stopListening();
+                return WidgetCatalogScreen();
+              }));
+              // Stop listening when navigate to Catalog Screen
+             detector.stopListening();
+            }
+          },
+          shakeSlopTimeMS: 1000);
+      detector.startListening();
+    } else if (state is DebugOptionEnabledState &&
+        state.debugOptionEnabled == false) {
+      // Stop listening when turn of debugOption
+      try{
+        detector.stopListening();
+      }catch(e){
+        
+      }
+   
+    }
+  }
+
   final ValueNotifier<int> _counter = ValueNotifier<int>(0);
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-        appBar: AppBar(
-          // leading: IconButton(
-          //   icon: const Icon(Icons.arrow_back, color: Colors.black),
-          //   onPressed: () => {},
-          // ),
-          title: Text(HatSpaceStrings.of(context).app_name),
-          centerTitle: true,
-        ),
-        body: Center(
-          child: Text(HatSpaceStrings.of(context).homePageViewTitle),
-        ),
-        bottomNavigationBar: ValueListenableBuilder<int>(
-          builder: (BuildContext context, int value, Widget? child) {
-            return BottomNavigationBar(
-              type: BottomNavigationBarType.fixed,
-              items: <BottomNavigationBarItem>[
-                BottomNavigationBarItem(
-                  icon: SvgPicture.asset(
-                    Assets.images.searchNormal,
-                    width: 24,
-                    height: 24,
-                    color: _counter.value == 0
-                        ? HSColor.secondary
-                        : HSColor.neutral4,
+    return BlocBuilder<AppConfigBloc, AppConfigState>(
+      //  TODO: variable just for testing - remove it when merge branch
+        builder: ((context, state) {
+          bool enableDebug = false;
+          if (state is DebugOptionEnabledState && state.debugOptionEnabled == true){
+            enableDebug = true;
+          }
+      onShakeToAction(context, state);
+      return Scaffold(
+          appBar: AppBar(
+            title: Text(HatSpaceStrings.of(context).app_name),
+            centerTitle: true,
+          ),
+          body: Center(
+              child: !enableDebug? Text(
+            HatSpaceStrings.of(context).homePageViewTitle,) : const Text("Debug Option Enabled")
```

Comment:
Do not change title base on debug option

Reply:
That's for testing purpose to demonstrate that debug_option_enabled has been gotten successfully.
Will remove it eventually

Reply:
Please remove in this PR